README.md

```diff
@@ -260,6 +264,22 @@ You can use watermarking by adding a new udf step in your metric:
     delayThreshold: 2 hours
 ```
 
+##### File Streaming Input
+Metorikku supports loading streaming Data from external storage systems. 
```

Comment:
Metorikku supports streaming over a file system as well.
Metorikku will process all new files added to the folder specified.